### ISSUE TITLE
CSS Comments

### ISSUE THREAD

**Issue by blackfalcon** [amzn/style-dictionary#195]
I noticed an issue with Style Dictionary when using comments to Sass and CSS. In Sass we get 

```
$color-type-theme-classic-light-link: #2774ae; // deprecated classic token color
```

In CSS we also get 

```
:root {
 --color-type-theme-classic-light-link: #2774ae; // deprecated classic token color
 --color-type-theme-classic-light-link-hover: #2774ae; // deprecated classic token color
}
```

This is not valid CSS and when using other post processors I get the following error

```
(node:85291) UnhandledPromiseRejectionWarning: CssSyntaxError: /Users/.../tokens/example/tokensBuild/TokenVariables.css:8:56: Unknown word
```

Also, this ONLY happens when there is an illegal comment as the last line. For example, when this is the variable set, there is no error. 

```
:root {
 --color-type-theme-classic-light-link: #2774ae; // deprecated classic token color
 --color-type-theme-classic-light-link-hover: #2774ae; // deprecated classic token color
 --color-type-disabled: #c8c9c7;
}
```

**Comment by dbanksdesign** [comment]
Interesting, you are correct I guess we have always been doing CSS comments wrong. Opened a PR to fix it: #196

**Comment by chazzmoney** [comment]
Fix in develop, will be in the next release
